src/domain/waste-balances/field-mappings.js

```diff
@@ -0,0 +1,43 @@
+import { PROCESSING_TYPES } from '#domain/summary-logs/meta-fields.js'
+import { EXPORTER_FIELD } from './constants.js'
+
+export const COMMON_FIELD = Object.freeze({
```

Comment:
Assuming these are the fields from the summary log, they should probably be defined there and reused?